src/renderers/shared/reconciler/__tests__/ReactIdentity-test.js

```diff
@@ -107,12 +107,12 @@ describe('ReactIdentity', function() {
     var Wrapper = React.createClass({
 
       render: function() {
-        var span1 = <span ref="span1" key={key} />;
-        var span2 = <span ref="span2" />;
+        var s1 = <span ref="span1" key={key} />;
```

Comment:
Ahh, I keep looking above but not below. This is fine then; you could also just inline them.